# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libgif/no-html-docs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

We don't need html docs and depend on imagemagick's convert, ...

--- giflib-5.2.2/doc/Makefile.vanilla	2025-08-03 20:55:31.822202842 +0200
+++ giflib-5.2.2/doc/Makefile	2025-08-03 20:54:50.422698568 +0200
@@ -15,7 +15,7 @@
 .adoc.html:
 	asciidoc $<
 
-all: allhtml manpages
+all: manpages
 
 # The distinction between XMLMAN and XMLINTERNAL is because
 # some pages shouldn't be installed as part of a binary package;
